README.md

```diff
@@ -132,6 +132,7 @@ readers and writers for the format.  The types are:
   - FLOAT: IEEE 32-bit floating point values
   - DOUBLE: IEEE 64-bit floating point values
   - BYTE_ARRAY: arbitrarily long byte arrays.
+  - FIXED_LEN_BYTE_ARRAY: fixed length byte arrays.
```

Comment:
Thanks for adding this!

It is weird that only `BYTE_ARRAY` and `FIXED_LEN_BYTE_ARRAY` end with a period. Better to remove them off for consistency.

Should we also mark INT96 as deprecated? cc @gszadovszky @shangxinli

Reply:
Yes, sure, we can mark `INT96` as deprecated.
It is unfortunate that we have documentation/specifications in multiple places and we usually forget about them. E.g. [parquet.apache.org](https://parquet.apache.org/docs/file-format/types/) still not contain `FIXED_LEN_BYTE_ARRAY`. We might create a jira about clearing up these. Even referencing the `parquet-format` repo from the apache site or generating the content from it?

Reply:
Sure. Give me some time to learn how the site works first.

Reply:
I agree that a single source of truth would be much more beneficial to first-time readers :)